ucastnici/6.8.2016/TomasNyvlt.md

```diff
@@ -0,0 +1 @@
+Issue5
```

Comment:
Tomáši, tak když se ti ten kurz nelíbí, tak mi to řekni!!!

(napiš tam něco víc!)